src/GitLinkTask/GitLink.targets

```diff
@@ -1,11 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <UsingTask TaskName="GitLinkTask.LinkProject" AssemblyFile="$(MSBuildThisFileDirectory)..\..\GitLinkTask.dll" />
+  <UsingTask TaskName="GitLink" AssemblyFile="$(MSBuildThisFileDirectory)GitLinkTask.dll" />
 
-    <Target Name="GitLinkProject" AfterTargets="CopyFilesToOutputDirectory">
-        <LinkProject
-            SolutionDirectory="$(SolutionDir)"
-            ProjectName="$(MSBuildProjectName)" />
-    </Target>
+  <Target Name="LinkSymbolsToRemoteGitRepo" AfterTargets="Compile">
```

Comment:
Great points, @dolly22. I'll add these.

BTW, no one has reviewed #111 yet, which is the first in the set of smaller PRs I was submitting.